docs/static_site/src/pages/get_started/osx_setup.md

```diff
@@ -110,13 +110,10 @@ the guide in [Math Library Selection](build_from_source#math-library-selection).
 ```bash
 rm -rf build
 mkdir -p build && cd build
-cmake -GNinja -C ../config.cmake ..
-cmake --build . --parallel 8
+cmake ..
+cmake --build .
 ```
 
-You can increase the `--parallel 8` argument to match the number of processor
```

Comment:
I think we can leave this as an option in the doc. Some users may not want to use all the processors for compiling.